mso/resource_ndo_schema_template_deploy.go

```diff
@@ -77,15 +77,43 @@ func resourceNDOSchemaTemplateDeploy() *schema.Resource {
 				Optional: true,
 				Default:  "always-deploy",
 			},
+
+			"site_ids": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of site IDs where template is deployed. If not provided, will be retrieved from API during undeploy",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+
+			"undeploy": &schema.Schema{
```

Comment:
Should we have an undeploy_all option like we added in [ansible](https://github.com/CiscoDevNet/ansible-mso/pull/728)? Then both integrations will have similar behaviour.
`site_ids` would be required on `undeploy` and ignored on `undeploy_all`